MoneyManagerModel/InitialData.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Diagnostics;
+
+namespace MoneyManagerModel
+{
+    public static class InitialData
+    {
+        public static List<User> 
+            Users { get; private set; }
+
+        public static List<Asset> 
+            Assets { get; private set; }
+
+        public static List<Category> 
+            Categories { get; private set; }
+
+        public static List<Transaction> 
+            Transactions { get; private set; }
+
+        static InitialData()
+        {
+            using (var sr 
+                = new StreamReader(
+                    $"{AppDomain.CurrentDomain.BaseDirectory}" +
+                    $"/initial_data.json"))
+            {
+                JsonSerializerOptions options 
+                    = new JsonSerializerOptions();
+                options.Converters.Add(
+                    new DateTimeConverter());
+
+                var root = JsonSerializer
+                    .Deserialize<Root>(
+                    sr.ReadToEnd(), 
+                    options);
+
+                Users = root.Users;
+                Assets = root.Assets;
+                Categories = root.Categories;
+                Transactions = root.Transactions;
+            }
+        }
+
+        private class Root
+        {
+            public List<User> 
+                Users { get; set; }
+
+            public List<Asset> 
+                Assets { get; set; }
+
+            public List<Category> 
+                Categories { get; set; }
+
+            public List<Transaction> 
+                Transactions { get; set; }
+        }
+
+        private class DateTimeConverter 
+            : JsonConverter<DateTime>
+        {
+            public override DateTime Read(
+                ref Utf8JsonReader reader, 
+                Type typeToConvert, 
+                JsonSerializerOptions options)
+            {
+                Debug.Assert(
+                    typeToConvert == typeof(DateTime));
+
+                var date = reader
+                    .GetString()
+                    .Split('.');
```

Comment:
'.' char could be a private field of the class since you use it in multiple places.